eve/main.yml

```diff
@@ -0,0 +1,69 @@
+version: "0.2"
+
+branches:
+  default:
+    stage: "pre-merge"
+
+stages:
+  pre-merge:
+    worker: &pod
+      type: kube_pod
+      path: eve/workers/zenko.yaml
+    steps:
+    - TriggerStages:
+        name: trigger all the tests
+        stage_names:
+        - helm_test_kube_1.9.6
+
+  helm_test_kube_1.9.6:
+    worker:
+      <<: *pod
+      service:
+        requests:
+          version: "1.9.6"
+        namespaces:
+        - "testNamespace"    # <<< the default namespace for that stage
+    steps:
+    - Git: &git_pull
+        name: git pull
+        repourl: "%(prop:git_reference)s"
+        mode: full
+        method: clobber
+        retryFetch: true
+        haltOnFailure: true
+    - ShellCommand: &install_tiller
+        name: install helm (tiller into kubernetes)
+        command: >-
+          helm init --wait
+        env:
+          TILLER_NAMESPACE: '%(prop:testNamespace)s'
+    - ShellCommand:
+        name: Install needed repo
+        command: >
+          helm repo add incubator http://storage.googleapis.com/kubernetes-charts-incubator
+        env:
+          TILLER_NAMESPACE: '%(prop:testNamespace)s'
+    - ShellCommand:
+        name: Retrieve dependency
+        command: >-
+           helm dep build charts/zenko
+        env:
+          TILLER_NAMESPACE: '%(prop:testNamespace)s'
+    - ShellCommand:
+        name: Install Zenko !
+        command: >-
+          helm upgrade zenko-test --namespace %(prop:testNamespace)s
+          --install charts/zenko --wait
+          --set prometheus.rbac.create=false
```

Comment:
The following will need rbac disabled as well
`
  --set zenko-queue.rbac.enabled=false
  --set redis-ha.rbac.create=false 
`

Reply:
All disabling of RBAC is a bit troublesome to me. I know it's likely required right now, but not something we can maintain forever: in the end, 'real' deployments will have RBAC enabled, and if all we test is with it disabled, bugs can remain uncaught all too easily.

Reply:
Oh yeah that is a really valid point. We should bring it up in the next Releng meeting.